lib/widgets/components/stats/leaderboard_section.dart

```diff
@@ -0,0 +1,76 @@
+import 'package:coffeecard/base/strings.dart';
+import 'package:coffeecard/base/style/text_styles.dart';
+import 'package:coffeecard/cubits/statistics/statistics_cubit.dart';
+import 'package:coffeecard/widgets/components/receipt/filter_bar.dart';
+import 'package:coffeecard/widgets/components/stats/leaderboard_entry.dart';
+import 'package:coffeecard/widgets/components/stats/statistics_dropdown.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+
+class LeaderboardSection extends StatelessWidget {
+  const LeaderboardSection({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return BlocBuilder<StatisticsCubit, StatisticsState>(
+      builder: (context, state) {
+        final int? userRank = state.getUserRank();
+        final bool userInLeaderboard =
+            userRank != null && userRank <= state.leaderboard.length;
+
+        return Column(
+          crossAxisAlignment: CrossAxisAlignment.stretch,
+          children: [
+            Padding(
+              padding: const EdgeInsets.fromLTRB(16, 0, 16, 16),
```

Comment:
Bottom padding should be 8

Reply:
i think the spacing between the bottom card and the leaderboard widget becomes too big (img 1)

![image](https://user-images.githubusercontent.com/39860858/154140538-2a9aaa52-cb55-4aa8-97d8-cdc69513c69d.png)

A solution could be to remove the Gap(24) in the stats_page (img 2)

![image](https://user-images.githubusercontent.com/39860858/154140645-6ee20c79-364f-47b4-bfe8-8431b2420755.png)

Reply:
Are we both talking about the padding below the `Leaderboards` title? I don't see how reducing that bottom padding would increase the space between the two sections

Reply:
Sure, I can approve of this change

Reply:
The cards need reduced padding (`CardBase` has a `dense` parameter; set that to `true`)